scanpy/plotting/_tools/scatterplots.py

```diff
@@ -55,7 +55,7 @@ def embedding(
     img_key: Optional[str] = None,
     crop_coord: Tuple[int, int, int, int] = None,
     alpha_img: float = 1.0,
-    scale_spot: float = 0.5,
+    scale_spot: float = 1.0,
```

Comment:
Just reuse size for that and update its docs like “if `img_key` is specified, `size` scales …”

Then you could `circles` to handle `size=None`.